test/WebJobs.Script.Tests/Microsoft.Azure.WebJobs.Script.WebHost.deps.json

```diff
@@ -21,12 +21,19 @@
           "Microsoft.AspNetCore.Mvc.NewtonsoftJson": "8.0.1",
           "Microsoft.Azure.AppService.Middleware.Functions": "1.5.5",
           "Microsoft.Azure.AppService.Proxy.Client": "2.3.20240307.67",
-          "Microsoft.Azure.Functions.PythonWorker": "4.34.0",
+          "Microsoft.Azure.Functions.DotNetIsolatedNativeHost": "1.0.12",
+          "Microsoft.Azure.Functions.JavaWorker": "2.18.1",
+          "Microsoft.Azure.Functions.NodeJsWorker": "3.10.1",
+          "Microsoft.Azure.Functions.Platform.Metrics.LinuxConsumption": "1.0.4",
```

Comment:
There were deps file changes, so I took the new deps file output and copied it into this baseline file. The only dependency change I made was to add a dependency to the new nuget package **Microsoft.Azure.Functions.Platform.Metrics.LinuxConsumption**